packages/codegen/src/codegen.ts

```diff
@@ -82,7 +101,7 @@ export async function generate({
     sourceFile.addImportDeclaration({
       namedImports: packageImports,
       moduleSpecifier: packageName,
-      leadingTrivia: `// Generated by Xata Codegen ${VERSION}. Please do not edit.\n`
+      leadingTrivia: existingCode ? undefined : `// Generated by Xata Codegen ${VERSION}. Please do not edit.\n`
```

Comment:
Non-blocking suggestion (or good potential follow up): Should we have comments making it super-clear the parts of the file they should / should not edit?

Reply:
Yes! That's why I'm removing it if it's being generated over, but I want to draft and see the parts that we would keep or not first before adding potentially other comments.